developers/index.html

```diff
@@ -44,12 +44,12 @@ <h3>
         </h3>
 
         <p>
-          The WebVR API is currently available in <a href="https://webvr.rocks/firefox">Firefox for Windows</a>, in
-          <a href="https://www.google.com/chrome/browser/mobile/index.html">Chrome 56+ for Android</a> (<a href="https://webvr.rocks/chrome_for_android#what_is_the_webvr_origin_trial">Origin Trial</a>),
-          <a href="../get-chrome/">Chrome Canary for Windows</a>,
-          <a href="https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webvr/">Microsoft Edge for Windows</a>,
-          <a href="http://developer.samsung.com/technical-doc/view.do?v=T000000202">Samsung Internet for Gear VR</a>,
-          <a href="https://www.supermedium.com">Supermedium for Windows</a>, and
+          The WebVR API is currently available in <a href="https://webvr.rocks/firefox">Firefox for Windows</a>; <a href="https://mixedreality.mozilla.org/firefox-reality/">Firefox Reality for Oculus Go, Daydream, and VIVEPORT</a>; 
+          <a href="https://www.google.com/chrome/browser/mobile/index.html">Chrome 56+ for Android</a> (<a href="https://webvr.rocks/chrome_for_android#what_is_the_webvr_origin_trial">Origin Trial</a>);
+          <a href="../get-chrome/">Chrome Canary for Windows</a>;
+          <a href="https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webvr/">Microsoft Edge for Windows</a>;
+          <a href="http://developer.samsung.com/technical-doc/view.do?v=T000000202">Samsung Internet for Gear VR</a>;
+          <a href="https://www.supermedium.com">Supermedium for Windows</a>; and
           <a href="https://nightly.mozilla.org/">Firefox Nightly builds</a> on macOS.
```

Comment:
let's change `Firefox Nightly builds on macOS` to `Firefox Nightly builds for macOS` for consistency